File: c++/01/01.cpp
#include <iostream>

int main() {
  int sum = 0;

  for (int i = 0; i < 1000; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      sum += i;
    }
  }

  std::cout << sum << "\n";

  return 0;
}

File: rust/01/01.rs
fn main() {
    let mut i = 1;
    let mut sum = 0;

    while i < 1000 {
        if i % 3 == 0 || i % 5 == 0 {
            sum += i;
        }

        i += 1;
    };

    println(fmt!("%d", sum));
}
